frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -282,6 +282,17 @@ class QuestionInner {
     return this.setCard(assoc(this.card(), "display", display));
   }
 
+  /**
+   * The cache_ttl of a question if it exists
+   */
```

Comment:
I'm not sure this comment brings more context than the method/property name themselves. Should we remove it?